x-pack/plugins/security_solution/server/routes/index.ts

```diff
@@ -171,11 +171,10 @@ export const initRoutes = (
       riskEnginePrivilegesRoute(router, getStartServices);
     }
   }
-  if (config.experimentalFeatures.entityAnalyticsAssetCriticalityEnabled) {
```

Comment:
Because it is a setting now, I think we need to enable all routes. @hop-dev can you confirm my assumption?
I added an extra check, `assertAdvancedSettingsEnabled`, that throws if the setting is disabled.

I will create a ticket also to check the settings on the UI side so we don't call the privileges API when we don't need to.